test/runtests.jl

```diff
@@ -27,13 +27,12 @@ include("awss3.jl") # creates `s3path_tests(config)`
         # We then run all tests with s3 directly.
 
         # We use multiple directories so that Minio can support versioning.
-        root = mktempdir()
-        dirs = [mkdir(joinpath(root, string(i))) for i in 1:12]
         port = 9005
-        minio_server = Minio.Server(dirs; address="localhost:$port")
+        minio_server = Minio.Server([mktempdir()]; address="localhost:$port")
```

Comment:
maybe this change is why you need to disable `s3_enable_versioning` on Minio?

Reply:
For whatever reason, this only seems to work on fairly old versions of min.io.  In more recent versions they are expecting these directories to be on physically distinct drives, and if not, it will crash when trying to start the server.  I tried getting this to work but could not.

Suggest we leave it like I now have it, and if you want to try fixing it after that'd be great, but I think it'd be best for everyone if you ensure it works on latest tagged min.io.

Reply:
Ok, makes sense